src/pps/pps.proto

```diff
@@ -940,6 +940,33 @@ message CreatePipelineTransaction {
   string effective_json = 3; // the effective spec: the result of merging the user JSON into the cluster defaults
 }
 
+message ProjectDefaults {
```

Comment:
I'm just curious what the plan is for this message; it's not a component of either the Request or Response messages.

Reply:
It’s not used directly, but is used in JSON-encoded form in the `project_defaults_json` field of the `GetProjectDefaultsResponse` and the `project_defaults_json` field of the `SetProjectDefaultsRequest`.